Migrations/Version20240701112200.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KimaiPlugin\AakSamlBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240701112200 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('DROP INDEX UNIQ_964F38AFFA0C224 ON kimai2_aak_saml_team_meta');
+        $this->addSql('ALTER TABLE
+          kimai2_aak_saml_team_meta
+        ADD
+          org_unit_id INT NOT NULL AFTER team_id,
+        ADD
+          dept_ids JSON NOT NULL COMMENT \'(DC2Type:json)\',
+        CHANGE
+          company_id company_id INT DEFAULT NULL,
+        CHANGE
+          company_name company_name VARCHAR(255) DEFAULT NULL,
+        CHANGE
+          division_id division_id INT DEFAULT NULL,
+        CHANGE
+          division_name division_name VARCHAR(255) DEFAULT NULL,
```

Comment:
Just out of curiosity: Did you format this yourself or can the `make:migration` command do this? It greatly improves the readability.

Reply:
Generated with the `--formatted` option
```
bin/console doctrine:migrations:diff --em=default --configuration=var/plugins/AakSamlBundle/Migrations/aak-saml.yaml --formatted
```